.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```

Comment:
Isn't to be ignored?

Reply:
I don't think so, look this repository -> https://github.com/vtex-apps/dreamstore